tests/nodes/primary_nodes/test_project.py

```diff
@@ -95,3 +99,510 @@ def test_integration_project(cript_api, simple_project_node):
 
     # ========= test delete =========
     delete_integration_node_helper(cript_api=cript_api, node_to_delete=simple_project_node)
+
+    ########################################################################
+
+
+########################################################################
+
+
+@pytest.mark.skip(reason="api")
+def test_update_project_change_or_reset_newly_made_materials(cript_api) -> None:
+    """
+    pytest nodes/primary_nodes/test_project.py::test_update_project_change_or_reset_newly_made_materials
+
+    """
+
+    epoch_time = int(time.time())
+    name_1 = f"my_proj_ali_{epoch_time}"
+    mat_1 = f"my_mat__{epoch_time}"
+    col_name = "031o0col"
+
+    # print("hello")
+
+    # url_path = f"/{node.node_type_snake_case}/"
+    url_path = "/project/"
+    create_payload = {"node": ["Project"], "name": name_1, "material": [{"uuid": "1809330c-31d2-4a80-af72-77b84070ee1d"}, {"uuid": "ea8f957c-b6e5-4668-b306-e0d6b0d05d9a"}]}
+
+    # print(cript_api._capsule_request(url_path=url_path, method="POST", data=json.dumps(create_payload)).json())
+
+    # quit()
+    try:
+        create_response = cript_api._capsule_request(url_path=url_path, method="POST", data=json.dumps(create_payload))
```

Comment:
for example, you can't have calls like this in the final test.

just calling cript_api.save(project) should be good enough.